lib/init/config-initializer.js

```diff
@@ -562,6 +564,13 @@ function promptUser() {
             initial: 0,
             choices: ["JavaScript", "YAML", "JSON"]
         },
+        {
+            type: "select",
+            name: "packageManager",
+            message: "Which package manager does your project use?",
+            initial: 0,
+            choices: ["npm", "yarn"]
+        },
```

Comment:
Should we ask this question always, or only when it's needed?

There are scenarios where nothing has to be installed, so the question seems unnecessary:

```
npx eslint --init
√ How would you like to use ESLint? · problems
√ What type of modules does your project use? · esm
√ Which framework does your project use? · none
√ Does your project use TypeScript? · No / Yes
√ Where does your code run? · browser
√ What format do you want your config file to be in? · JSON
√ Which package manager does your project use? · yarn
Successfully created .eslintrc.json file in ...
```

If we are not anticipating that the npm/yarn answer will be used for something else, should we maybe ask this after the user answers "Yes" on "Would you like to install them?", or alternatively change the list of answers to something like: "No"/"Yes, with npm"/"Yes, with yarn". 

Thoughts?

Reply:
Yes, this ques should be asked after `Would you like to install them?`